pkgs/by-name/re/resources/package.nix

```diff
@@ -48,8 +51,16 @@ stdenv.mkDerivation (finalAttrs: {
     glib
     gtk4
     libadwaita
+    dmidecode
+    polkit
+    systemd
   ];
 
+  postPatch = ''
+    substituteInPlace src/utils/memory.rs \
+      --replace '"dmidecode"' '"${dmidecode}/bin/dmidecode"'
```

Comment:
Wouldn't the same thing also be necessary for `pkexec` if its not in path?
(https://github.com/nokyan/resources/blob/main/src/utils/memory.rs#L185)

Reply:
Good catch. I thought it should also be patched.

Reply:
And maybe it doesn't need to be in the buildInputs? What is systemd for?

Reply:
https://github.com/nokyan/resources/blob/504398b6de2f37fb127217ce4c9850318f42bb5d/src/utils/cpu.rs#L66 This should also be patched.

Reply:
I've taken the changes from the other package, but I assume systemd is for something udev related? Upstream references it as a dependency, but I've not looked further.

Agree on the other two changes, will add those to this fixup.

Reply:
I've pushed a change adding util-linux for lscpu, and dropping polkit and systemd as:
- makes no sense to pull polkit in if it's not enabled on the system, and the software should fail if it's not present in the specified path (see anything else that calls pkexec in nixpkgs)
- only references to systemd in upstream don't actually make any sense: https://github.com/search?q=repo%3Anokyan%2Fresources%20systemd&type=code